src/app/modules/common-ui/input-error-text/input-error-text.component.ts

```diff
@@ -0,0 +1,33 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { AbstractControl } from '@angular/forms';
+
+@Component({
+  selector: 'app-input-error-text',
+  templateUrl: './input-error-text.component.html',
+  styleUrls: ['./input-error-text.component.css'],
+  changeDetection: ChangeDetectionStrategy.Default
```

Comment:
You already have a . Default change strategy remove it.
If you wish to use OnPush strategy use  ChangeDetectionStrategy.OnPush

Reply:
Deleted the whole component